src/OrchardCore.Modules/OrchardCore.Lists/Models/ContainedItemOptions.cs

```diff
@@ -0,0 +1,18 @@
+using OrchardCore.Contents.ViewModels;
+
+namespace OrchardCore.Lists.Models
+{
+    public enum ContentsStatus
+    {
+        None,
+        Draft,
+        Published,
+        Owner
+    }
+    public class ContainedItemOptions
+    {
+        public string DisplayText { get; set; }
+
```

Comment:
Fölös sor, a class elég viszont kellene.

Reply:
Tovább is kellene a class elé.

Reply:
ok